.github/workflows/hourly.yml

```
@@ -22,6 +22,8 @@ jobs:
       run: nginx -c /etc/nginx/nginx.conf
     - name: triage
       run: LD_PRELOAD=libnss_wrapper.so enarx-triage
+    - name: auto-request
+      run: LD_PRELOAD=libnss_wrapper.so enarx-auto-request
```

```suggestion
      run: LD_PRELOAD=libnss_wrapper.so enarx-pr-request
```

We know it is `auto`, it is a bot. :) What we don't know is what kind of objects it operates on.